### PR TITLE
Check for existence of $_GET keys

### PR DESCRIPTION
**Pull request by LukasReschke**
Otherwise PHP errors are thrown in the error log.

@MorrisJobke Your code - please review - THX.

### CODE REVIEW

File: apps/files/ajax/rename.php [hunk 1]
```diff
@@ -30,13 +30,13 @@
 	\OC::$server->getL10N('files')
 );
 $result = $files->rename(
-	$_GET["dir"],
-	$_GET["file"],
-	$_GET["newname"]
+	isset($_GET['dir']) ? $_GET['dir'] : '',
```
Comment (MorrisJobke) [inline apps/files/ajax/rename.php]: I would rather check that in advance and then return a failure, as an empty filename is useless for a file rename. ;)

Comment (LukasReschke) [inline apps/files/ajax/rename.php]: Well - the failure will already be catched within the rename function - didn't want to duplicate too many code.

Comment (MorrisJobke) [inline apps/files/ajax/rename.php]: Really? Okay... I need to read the code

**Comment by MorrisJobke** [comment]
You were right. :+1: rename still works

**Comment by nickvergessen** [comment]
:+1:

**Pull request merged**